README.md

```diff
@@ -1,26 +1,27 @@
 Key Recovery Service
 ====================
-The key recovery service dispenses xpubs (Bitcoin public keys) to anyone who requests one for use as a cold key in multisig wallets.
+The key recovery service dispenses BIP32 compliant extended public keys (xpubs) to anyone who requests one for use as a cold key in multisig wallets. These keys are compatible with *most* cryptocurrencies using the Secp256k1 elliptic curve.
```

Comment:
It's not exactly easy to do cross-compare BitGo supported coins and secp256k1 coins.  Maybe point them to a document that lists the coins that work with this

Reply:
I don't know of any list that we could link to, but I added a link to the BIP32 specification.